### PR TITLE
Relay Bid Adapter : Initial Release

### PR DESCRIPTION
**Pull request by jcswart**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [x] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes

- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

Email of adapter's maintainer: [relay@kevel.co](mailto:relay@kevel.co)

Add the Relay Bid Adapter

## Other information

Doc PR: https://github.com/prebid/prebid.github.io/pull/4720

### CODE REVIEW

**Comment by ChrisHuie** [comment]
@jcswart can you please pull in recent commits on master. Hopefully that fixes the circleci build issue

**Comment by jcswart** [comment]
@ChrisHuie All fixed up, thank you.

**Comment by jcswart** [comment]
Any news on this @mmoschovas or @ChrisHuie ?

File: modules/relayBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+      if (missingGdpr) {
+        syncUrl.searchParams.set('gdpr', Number(gdprConsent.gdprApplies))
+        sync.url = syncUrl.toString();
+      }
+      if (missingGdprConsent) {
+        syncUrl.searchParams.set('gdpr_consent', gdprConsent.consentString);
+        sync.url = syncUrl.toString();
+      }
+      if (syncOptions.iframeEnabled && sync.type === 'iframe') { syncs.push(sync); }
+      if (syncOptions.pixelEnabled && sync.type === 'image') { syncs.push(sync); }
```
Comment (mmoschovas) [inline modules/relayBidAdapter.js:72]: Can this be an else if? Is there any reason to potentially allow both if image and iframe are enabled

Comment (jcswart) [reply modules/relayBidAdapter.js]: This change has been implemented in the latest commit.

**Review by mmoschovas — APPROVED** [review]

**Pull request merged**